### PR TITLE
simplifying steering to a single line with move_toward

### PR DESCRIPTION
**Pull request by LaBodilsen**
Replaced:
```
	if steer_target < steer_angle:
		steer_angle -= STEER_SPEED * delta
		if steer_target > steer_angle:
			steer_angle = steer_target
	elif steer_target > steer_angle:
		steer_angle += STEER_SPEED * delta
		if steer_target < steer_angle:
			steer_angle = steer_target
	
	steering = steer_angle
```
with
`	steering = move_toward(steering, steer_target, STEER_SPEED * delta)`

### CODE REVIEW

**Comment by aaronfranke** [comment]
I think it would be better to use `move_toward` instead. If you change `lerp` to `move_toward`, add `* delta` in the last part, and set `STEER_SPEED` back to 1, the behavior is the same as before.

Also, `steer_angle` can be removed since it's not used anymore.

**Comment by LaBodilsen** [comment]
Clever to use the move_towards function instead. 

i agree that it would be the better solution.  should make a new pull request for this, and revoke this one?

File: 3d/truck_town/vehicle.gd [hunk 1]
```diff
@@ -27,13 +26,4 @@ func _physics_process(delta):
...
-		steer_angle -= STEER_SPEED * delta
-		if steer_target > steer_angle:
-			steer_angle = steer_target
-	elif steer_target > steer_angle:
-		steer_angle += STEER_SPEED * delta
-		if steer_target < steer_angle:
-			steer_angle = steer_target
-	
-	steering = steer_angle
+	steering = move_toward(steering, steer_target, STEER_SPEED*delta)
```
Comment (aaronfranke) [inline 3d/truck_town/vehicle.gd]: There should be spaces around `*`

Comment (LaBodilsen) [reply 3d/truck_town/vehicle.gd]: You are of course correct.  

That is now implemented in the request.

**Comment by aaronfranke** [comment]
Squashing-and-merging, thanks!

**Pull request merged**